app/views/form/check_answers.html.erb

```diff
@@ -17,10 +17,16 @@
     <% end %>
     <%= display_answered_questions_summary(subsection, @case_log, current_user) %>
 
-    <%= render partial: "form/check_answers_summary_list", locals: {
-      subsection:,
-      case_log: @case_log,
-    } %>
+    <% if subsection.id == "household_characteristics" %>
```

Comment:
We should probably avoid hardcoding specific section names here, will make the code brittle. Could we instead add the key everywhere or infer based on it's absence such that we can do this generically?

Reply:
I'd considered that but this is the only subsection that uses the summary list design for the check answers page (so I scrapped trying to generalise it since we probably wouldn't need it for any others) and we're doing the same kind of check for the setup section just a few lines above. Happy to try and generalise if you think it would be needed 😄

Reply:
My thinking is that "Setup" is a legit special case that behaves differently from everything else whereas this just incidently is. i.e. I think you could reasonably expect to add this functionality to any arbitrary set of questions in any section, so I probably would genericise it, but I agree we don't strictly need to.

Reply:
Would it be following the principle of "You ain't gonna need it" (YAGNI) if we were to generalise this only if a need for it actually arises? Asking because I'm not 100% sure 😄